docs/_index.md

```diff
@@ -4,97 +4,63 @@ meta_desc: Provides an overview of the Buildkite Provider for Pulumi.
 layout: overview
 ---
 
-The Buildkite provider for Pulumi can be used to manage resources in
-the [Buildkite](https://buildkite.com) CI/CD platform. The Buildkite
-provider must be configured with credentials to deploy and update
-resources.
+The Buildkite provider for Pulumi can be used to provision resources for [Buildkite](http://buildkite.com).
 
 ## Example
 
-{{< chooser language "typescript,python,go" >}}
-{{% choosable language typescript %}}
+{{< chooser language "javascript,typescript,python,csharp" >}}
 
-```typescript
-import * as pulumi from "@pulumi/pulumi";
-import * as buildkite from "@grapl/pulumi-buildkite";
-
-new buildkite.Pipeline(
-    "testing-pipeline",
-    {
-        name: "testing-pipeline",
-        repository: "https://github.com/grapl-security/pulumi-buildkite",
-        description: ":pulumi::buildkite::nodejs: A pipeline created to test the pulumi-buildkite provider",
-        defaultBranch: "main",
-        steps: `
-        steps:
-          - label: "Hello World"
-            command: echo 'Hello World'
-        `
-    }
-);
+{{% choosable language javascript %}}
+
+```javascript
+const buildkite = require("@pulumiverse/pulumi-buildkite")
 
+const agentToken = new buildkite.agent.AgentToken('token', {
+    description: 'agent token',
+})
 ```
 
-{{% /choosable %}}
-{{% choosable language python %}}
+{{% /choosable %}} {{% choosable language typescript %}}
 
-```python
-import pulumi
-import pulumi_buildkite as buildkite
-
-def main() -> None:
-    buildkite.Pipeline(
-        "testing-pipeline",
-        name="testing-pipeline",
-        repository="https://github.com/grapl-security/pulumi-buildkite",
-        description=":pulumi::buildkite::python: A pipeline created to test the pulumi-buildkite provider",
-        default_branch="main",
-        steps="""
-        steps:
-          - label: "Hello World"
-            command: echo 'Hello World'
-        """
-    )
-
-if __name__ == "__main__":
-    main()
+```typescript
+import * as buildkite from "@pulumiverse/pulumi-buildkite";
 
+const agentToken = new buildkite.agent.AgentToken('token', {
+    description: 'agent token',
+})
 ```
 
-{{% /choosable %}}
-{{% choosable language go %}}
+{{% /choosable %}} {{% choosable language python %}}
 
-```go
-package main
+```python
+import pulumi_buildkite_bridge as buildkite
```

Comment:
Isn't the correct Python package name `pulumiverse_buildkite` (see `resources.go`)?

Reply:
thank you - i seem to have missed to adapt all examples in the docs - fixed.